run/sense2john.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import xml.etree.ElementTree as ET
+
+def process_file(filename):
+    try:
+        fd = open(filename, "rb")
+    except IOError:
+        e = sys.exc_info()[1]
+        sys.stderr.write("%s\n" % str(e))
+        return
+
+    tree = ET.parse(filename)
+    root = tree.getroot()
+
+    for user in root.findall('system/user'):
+        username = user.find('name').text
+        if user.find('bcrypt-hash') is not None:
+            sys.stdout.write("%s:%s\n" % (username, user.find('bcrypt-hash').text))
+        elif user.find('md5-hash') is not None:
+            sys.stdout.write("%s:%s\n" % (username, user.find('md5-hash').text))
+        elif user.find('password') is not None:
+            sys.stdout.write("%s:%s\n" % (username, user.find('password').text))
```

Comment:
Is it possible that more than one hash type would be present for a user? If so, the script might want to output multiple lines per user, one per hash. I think this would be achieved by simply using `if` in place of `elif`.